gno.land/pkg/sdk/vm/builtins.go

```diff
@@ -34,10 +44,18 @@ func (vm *VMKeeper) initBuiltinPackagesAndTypes(store gno.Store) {
 			PkgPath: "gno.land/r/stdlibs/" + pkgPath,
 			// PkgPath: pkgPath,
 			Output: os.Stdout,
-			Store:  newStore,
+			Store:  store,
 		})
 		defer m2.Release()
-		return m2.RunMemPackage(memPkg, true)
+		pn, pv = m2.RunMemPackage(memPkg, true)
+		// XXX
+		if false {
+			fmt.Println(colors.Red("VMKeeper.init.getPackage ========================================= before wrote"))
+			store.Print()
+			fmt.Println(colors.Red("VMKeeper.init.getPackage ========================================= before wrote end"))
+		}
+		store.Write() // XXX XXX XXX or flush?
```

Comment:
essential change #1; the saving must be persisted to the underlying store, so that it can be picked up by a subsequent call that fetches it, but requires essential change #2 too